Assets/MRTK/Core/Providers/Hands/HandBounds.cs

```diff
@@ -17,6 +17,11 @@ public class HandBounds : MonoBehaviour, IMixedRealitySourceStateHandler, IMixed
         /// Accessor for the bounds associated with a handedness.
```

Comment:
Nit: Update the comments for this to mention that these are global bounds (I understand that changing the name would be tricky because of API surface changes) but doccomment updates are fine.

Reply:
Ok. I didn't know if changing the docstring would be "bad" with respect to the API surface. Good to know we can update them.